posts/2020-12-18-microprofile-long-running-actions-beta.adoc

```diff
@@ -281,8 +281,8 @@ Flight Booking compensated with LRA with id: http://localhost:9080/lrac/lra-coor
 ----
 These messages show the successful start of the LRA but since the business logic failed and the method returned an error response, the compensate method is automatically called and run.
 
-=== Extending the example
-While an LRA is useful for a single service, it is more common to have multiple services in an LRA. We can extend the example by having a service, BookHoliday, call the `BookFlight` service and another new service called `BookHotel.`
+=== Configuring an LRA with multiple participants
+While an LRA is useful for a single service, it is more common to have multiple services in an LRA. In the following examples, the `BookHoliday` service, calls the `BookFlight` service and another new service called `BookHotel.`
```

Comment:
"In the following examples," 
It is all part of the same example rather than multiple new examples which that seems to suggest.